tests/unittests/data/sample_to_pickle.py

```diff
@@ -0,0 +1,34 @@
+"""A small script that 'pickles' a PWA-formatted pandas.DataFrame."""
```

Comment:
I don't see the value for using this in a test (binary data)

Reply:
I'll tell you even worse: the `pkl` file it produces is used in an earlier commit... :grimacing:

Reply:
Yeah I also noted it there. Or what do you mean with earlier commit? Just a commit from this PR right?

Reply:
https://github.com/ComPWA/pycompwa/pull/78/commits/e3b93d7c5e03be4c8a155c7dfa69869afa0d0dc8